### DISCUSSION TITLE
privacy enhanced mode

### DISCUSSION THREAD

**Discussion by Nypheena** [sandboxie-plus/Sandboxie#1806] [Documents and notes]
so I just downloaded 1.0.8 and I figured I'd try privacy enhanced mode, but most programs seem to have major issues with it as they can't even detect the folders, I figured it'd work by only seeing whats in the sandbox but still access the folder path, if I make it readable it'll still see the host content but thats not what I want, I may be misunderstanding how this all works and would love a bit of guidance

**Comment by DavidXanatos** [comment]
It works by hiding and blocking access by default to everything but
C:\Windows
C:\Program Files
C:\Program Files (x86)

for access to any other path you must set this in the access settings of the box

**Comment by Nypheena** [comment]
when using winscp even if I add the drive as box only I get "Drive C not ready" I have to give it read access to host or it won't work and not sure how to get past that

**Comment by Nypheena** [comment]
it seems a big issue for games, even if I add box only access to all drives and even the users folder it won't work to save anything

**Comment by DavidXanatos** [comment]
so you are saying it can not access the boxed location eider?
or only the host location, <- that's expected

have you tries creating the folders manually in the box?

**Comment by TFWol** [comment]
I've been very confused by the Privacy Mode too, but mostly due to a hard time finding documentation and the wording of things (especially the debug tracer paths vs what should be entered in Resource Access to remedy). 

My attempts to learn by testing my own scenarios haven't been very fruitful; after hours of changing settings and finding a stable solution, I end up needing to rebuild after an update reveals a setting somewhere was working only because of a glitch. My understanding of things are pretty muddied now.

I've jumped around the changelogs and browse discussions to try and understand when something isn't documented, but that's pretty daunting and I often can't find exactly what I aim for. 
I've found #1076 helpful for getting my bearings on 'known broken' to avoid going down some rabbit holes.

One thing, added a couple of days ago, that helped immensely was the wording change from **Boxed Only** to "Box Only (Write Only)" https://github.com/sandboxie-plus/Sandboxie/blob/a20dab676043ea7a515a14a78001a0cfc80e78f8/SandboxiePlus/SandMan/sandman_it.ts#L380 

~I definitely misunderstood the original as "sandboxed program reads and writes to this sandboxed folder". That would explain the bizarre behavior I was seeing, but also confusing when I can read anyways (until one of the recent patches).~
Nope, I understood it correctly. It looks like this version of Sandboxie fixed whatever the issue was after I installed on it on my test computers.

I think having example scenarios for popular applications (Firefox, Discord, etc.) using Privacy Enhanced Mode would help. 
Something like a Firefox installation along with the recommended entries in **Resource Access Rules**, etc.

Great work with what the team is doing with the project all things considered. There are a lot of moving parts.


Fun fact:
A few months ago, your nostalgic Norman from Mighty Max picture on Patreon is what caught my eye and somehow led me to notice sandboxie was being updated.

**Comment by StoneMoe** [comment]
lots of my daily use programs refused to start with privacy mode, mostly like due to strictly resource access limit.

maybe `read as empty folder/registry value and writeable` is better than ` unsandboxed location access denied`?

**Comment by TigerCubDen** [comment]
I consider Box Only (Write Only) instead of block is a better way.

**Comment by e-t-l** [comment]
> I think having example scenarios for popular applications (Firefox, Discord, etc.) using Privacy Enhanced Mode would help.

Did anyone ever put together something like this? Would be a great way for new users to get started with something useful and functional, and also be able to examine the settings to get a better understanding of how Sandboxie works
